src/rabbit_variable_queue.erl

```diff
@@ -710,7 +710,7 @@ ack([], State) ->
 ack([SeqId], State) ->
     case remove_pending_ack(true, SeqId, State) of
         {none, _} ->
-            State;
+            { [], State};
```

Comment:
Thanks for finding this. Could you remove the leading whitespace character before `[]`? Thanks.

Reply:
I have done